src/firewheel_repo_linux/linux/INSTALL/tasks.yml

```diff
@@ -0,0 +1,60 @@
+---
+- name: Check if Linux Base Objects is already installed
+  ansible.builtin.stat:
+    path: "{{ install_flag }}"
+  register: flag_check
+
+- name: Fail if already installed
+  ansible.builtin.fail:
+    msg: "linux.base_objects is already installed!"
+  when: flag_check.stat.exists
```

Comment:
None of this is needed. FIREWHEEL will take care of any checks if the MC is already installed and prompt the user. See the example template for `tasks.yml` https://github.com/sandialabs/firewheel/pull/113/files#diff-29860a988b4a753ae02ed06cecbbcb60361745a343b322275dfd7a9403d41a8a